mock/src/main/java/feign/mock/MockClient.java

```diff
@@ -238,7 +236,8 @@ public Request verifyOne(HttpMethod method, String url) {
 
     List<Request> result = requests.get(requestKey);
     if (result.size() != times) {
-      throw new VerificationAssertionError("Wanted: '%s' to be invoked: '%s' times but got: '%s'!",
+      throw new VerificationAssertionError(
+          "Wanted: '%s' to be invoked: '%s' times but got: '%s'!",
```

Comment:
I still see a bunch of `No-Op` changes...

Would be nice to reduce the number of thoose